### PR TITLE
Implement av stack orphan command

### PR DESCRIPTION
**Pull request by Konboi**
Issue: https://github.com/aviator-co/av/issues/243

I tried to implement `av stack orphan` command.

### CODE REVIEW

File: cmd/av/stack_orphan.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			Upstream: upstream,
+			Branch:   branchName,
+		})
+		if err != nil {
+			return fmt.Errorf("failed to run git rebase")
+		}
+
+		if output.ExitCode != 0 {
+			_, _ = fmt.Fprint(os.Stderr,
+				colors.Failure("ERROR: "),
```
Comment (Konboi) [inline cmd/av/stack_orphan.go:66]: Referred https://github.com/aviator-co/av/blob/61844c50acbc15dc0cf153e8b18d16aa196c1a6d/internal/actions/reparent.go#L197-L205

File: cmd/av/stack_orphan.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+		if err != nil {
+			return fmt.Errorf("failed to get hash of parent branch: %s", currentBranch.Parent.Name)
+		}
+
+		upstream := currentBranch.Parent.Name
+		if currentBranch.Parent.Trunk {
+			upstream = "remotes/origin/" + currentBranch.Parent.Name
+		}
+
+		output, err := repo.Rebase(git.RebaseOpts{
```
Comment (Konboi) [inline cmd/av/stack_orphan.go:55]: Honestly, I'm not confident that this is the correct way to do a rebase...

**Comment by draftcode** [comment]
Hm. I think I didn't think through the behavior.

What this command should do would be:

1. Find the branch to be orphaned (and the child branches of that branch).
2. Delete the branch metadata from the db (See https://github.com/aviator-co/av/blob/master/internal/actions/tidy.go#L34-L41).

In other words, this command should just delete the av-cli's branch metadata.

**Comment by Konboi** [comment]
Thank you for your reviewing. I'll fix it

**Comment by Konboi** [comment]
I fixed. Is this behaving as you intended? https://github.com/aviator-co/av/pull/258#issuecomment-2064698440

**Pull request merged**